From 85892affca7dd3b56a4d29423a4bbe3d42b9314e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Lureau?= <marcandre.lureau@gmail.com>
Date: Thu, 4 Jul 2013 16:30:22 +0200
Subject: [PATCH] data/remote-viewer.desktop: Fix missing trailing ;

data/remote-viewer.desktop: error: value
"x-scheme-handler/spice;application/x-virt-viewer" for string list key
"MimeType" in group "Desktop Entry" does not have a semicolon (';') as
trailing character
---
 data/remote-viewer.desktop.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/remote-viewer.desktop.in b/data/remote-viewer.desktop.in
index 7fdfd47..115875d 100644
--- a/data/remote-viewer.desktop.in
+++ b/data/remote-viewer.desktop.in
@@ -4,6 +4,6 @@ _Comment=Access remote desktops
 Exec=remote-viewer %u
 Type=Application
 Terminal=false
-MimeType=x-scheme-handler/spice;application/x-virt-viewer
+MimeType=x-scheme-handler/spice;application/x-virt-viewer;
 StartupNotify=true
 Categories=GNOME;GTK;Network;RemoteAccess;
